docs/aks/README.md

```diff
@@ -351,65 +322,68 @@ helm repo update
 
 ```
 
-## Install Linkerd Service Mesh into the cluster
+### Install AAD Pod Identity for the application
 
-Download the Linkerd v2 CLI:
+Change directories to the `docs/aks` folder and make the `aad-podid.sh` script executable. Running this shell script will deploy AAD Pod Identity to your cluster and assign a Managed Identity.
+
+>NOTE: The second command below has a `.` then a space followed by `./aad-podid.sh ...` this is so the exported variables in the script persist after the script ends in the uder interactive shell
 
 ```bash
 
-# macOS and linux/WSL
+export MI_Name=${He_Name}-mi
 
-curl -sL https://run.linkerd.io/install | sh
-export PATH=$PATH:$HOME/.linkerd2/bin
+cd $REPO_ROOT/docs/aks
+sudo chmod +x aad-podid.sh
+
+. ./aad-podid.sh -a ${He_AKS_Name} -r ${He_App_RG} -m ${MI_Name}
+
+cd $REPO_ROOT
+./saveenv.sh
 
 ```
 
-Install the Linkerd control plane in the linkerd namespace:
+The last line of the output will explain the proper label annotation needed when deploying the application. This will be needed later during the application install
 
 ```bash
 
-linkerd install | kubectl apply -f -
+echo $MI_Name
 
 ```
 
-Validate the install with:
+### Set Keyvault Policy for MI User
 
 ```bash
 
-linkerd check
+az keyvault set-policy -n ${He_Name} --object-id ${MI_PrincID} --secret-permissions get list --key-permissions get list --certificate-permissions get list
```

Comment:
I think all we need is secret permission - I think key permission can be removed

Reply:
Might need to be removed from the aad-podid.sh script as well.